packages/lib/src/vis/scatter/ScatterVis.tsx

```diff
@@ -106,7 +137,25 @@ function ScatterVis(props: Props) {
           invertColorMap={invertColorMap}
           size={size}
           onClick={onPointClick}
+          onPointerEnter={onPointerEnter}
+          onPointerOut={() => hideTooltip()}
         />
+        <TooltipOverlay {...{ tooltipOpen, tooltipTop, tooltipLeft }}>
+          {tooltipIndex && (
+            <>
+              <span>
+                {`${abscissaLabel ?? 'x'} = ${formatTooltipVal(
+                  abscissas[tooltipIndex]
+                )}, ${ordinateLabel ?? 'y'} = ${formatTooltipVal(
+                  ordinates[tooltipIndex]
+                )}`}
+              </span>
+              <div className={styles.tooltipValue}>
+                <strong>{formatTooltipVal(dataArray.get(tooltipIndex))}</strong>
+              </div>
+            </>
+          )}
+        </TooltipOverlay>
```

Comment:
`LineVis` and `HeatmapVis` have a `renderTooltip` prop to tune this default tooltip. Do you think we should add it to `ScatterVis` as well ?

If yes, what arguments should it take ? Coordinates of the mouse, data abscissa, data ordinate, data value and index ?

Reply:
Hmm neh, no point until we find a use case (which will then help determine what arguments are needed).